aa_stripe/signals.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 import django.dispatch
 
-stripe_charge_succeeded = django.dispatch.Signal(providing_args=["instance"])
-stripe_charge_card_exception = django.dispatch.Signal(providing_args=["instance", "exception"])
-stripe_charge_refunded = django.dispatch.Signal(providing_args=["instance"])
+stripe_charge_succeeded = django.dispatch.Signal()
+stripe_charge_card_exception = django.dispatch.Signal()
+stripe_charge_refunded = django.dispatch.Signal()
```

Comment:
A bit of change. Is this working with 3.x too?

Reply:
The `providing_args` was a no-op in 2.2 ("This is purely documentational, however, as there is nothing that checks that the signal actually provides these arguments to its listeners."), and in 3.2: absent from the docs and deprecated in code.

Reply:
But thanks for looking closely! :)